utils/plotting_utils.R

```diff
@@ -0,0 +1,65 @@
+
+draw_confusion_matrix <- function(cm, c1_name, c2_name, title="CONFUSION MATRIX") {
+
+    ### taken from https://stackoverflow.com/questions/23891140/r-how-to-visualize-confusion-matrix-using-the-caret-package
+
+    total <- sum(cm$table)
+    res <- as.numeric(cm$table)
+
+    # Generate color gradients. Palettes come from RColorBrewer.
+    greenPalette <- c("#F7FCF5","#E5F5E0","#C7E9C0","#A1D99B","#74C476","#41AB5D","#238B45","#006D2C","#00441B")
```

Comment:
```suggestion
    greenPalette <- c(
    "#F7FCF5","#E5F5E0","#C7E9C0",
    "#A1D99B","#74C476","#41AB5D",
    "#238B45","#006D2C","#00441B"
    )
```
I know this was copied, but just wanted to add more whitespace to this.